library/custom_template/ajax_code.php

```diff
@@ -171,9 +170,9 @@
         echo "</span></li>";
     }
     if (AclMain::aclCheckCore('nationnotes', 'nn_configure') && $templateid) {
-        echo "<li style='cursor:pointer'><span onclick='add_item()'>" . htmlspecialchars(xl('Click to add new components'), ENT_QUOTES);
+        echo "<li class='bg-dark text-light' style='cursor:pointer'><span class='bg-dark text-light' onclick='add_item()'>" . htmlspecialchars(xl('Click to add new components'), ENT_QUOTES);
         echo "</span><div id='new_item' style='display:none' class='w-100'>";
-        echo "<textarea name='item' id='item' class='w-100'></textarea><br />";
-        echo "<input type='button' name='save' value='" . htmlspecialchars(xl('Save'), ENT_QUOTES) . "' onclick='save_item()'><input type='button' name='cancel' value='" . htmlspecialchars(xl('Cancel'), ENT_QUOTES) . "' onclick=cancel_item('" . htmlspecialchars($row['cl_list_slno'], ENT_QUOTES) . "')></div></li>";
+        echo "<textarea name='item' id='item' class='w-100 bg-dark text-light'></textarea><br />";
+        echo "<input type='button' name='save' value='" . htmlspecialchars(xl('Save'), ENT_QUOTES) . "' onclick='save_item()'><input type='button' name='cancel' value='" . htmlspecialchars(xl('Cancel'), ENT_QUOTES) . "' onclick=cancel_item('" . attr_js($row['cl_list_slno'] ?? '') . "')></div></li>";
```

Comment:
should drop the outside `'` around the attr_js call
(nice to get to provide some escaping input every once i awhile to feel useful :) )

Reply:
it's fine Brady catch as catch can!:)
Maybe @stephenwaite can add this to his PR though I disagree it is needed. Again, inspect it in browser.

Reply:
@sjpadgett , Current codebase doesn't have the `'` in all the calls and have not seen issue with this yet. Is there a use case where this breaks?

Reply:
Last issue I had was in one of my modules. I believe it was ringcentral but no matter. In this use case I suppose it doesn't matter but in the case where passing a string or an int was my issue I believe. Still either way it will parse and render the same up to the browser. I don't have an explanation for that!
I'll use it the way you intended though.

Reply:
Sounds good. Note in the special case of an int, attr_js will not add an `'` (json_encode does not add surrounding quote to int), which would be a reason not to also hard-code quotes around this function (since would then convert a int to a string).